src/Rings/AbelianClosure.jl

```diff
@@ -1084,6 +1119,23 @@ Base.conj(elm::QQAbFieldElem) = elm^QQAbAutomorphism(-1)
 
 Base.isreal(elm::QQAbFieldElem) = conj(elm) == elm
 
+# compare real `QQAbFieldElem`s
+function _isless_via_qqbar(a, b)
+  F = QQBarField()
+  return Base.isless(F(a), F(b))
+end
```

Comment:
Could also use `QQBarFieldElem` (or leave it as is)
```suggestion
_isless_via_qqbar(a, b) = Base.isless(QQBarFieldElem(a), QQBarFieldElem(b))
```